### PR TITLE
Ntd: fix naming conventions, expose ridership tables as mart

### PR DESCRIPTION
**Pull request by charlie-costanzo**
# Description

We initially brought in NTD tables as MVPs and have since realized we could improve naming conventions. This PR updates naming and exposes ridership staging tables as mart.

## Type of change
- [x] New feature

## How has this been tested?
Screenshot 2024-12-16 at 4 10 41 PM

## Post-merge follow-ups
- [x] Actions required (specified below)
UPDATE ANY REFERENCES TO THESE TABLES!! and observe for expected behavior

### CODE REVIEW

File: warehouse/models/mart/safety_and_security/_mart_safety_and_security.yml
Comment (erikamov) [inline warehouse/models/mart/safety_and_security/_mart_safety_and_security.yml:1]: I think would be nice to show that are NTD data, using NTD on the group name like: `_mart_ntd_safety_and_security`

File: warehouse/models/mart/ridership/_mart_ridership.yml
Comment (erikamov) [inline warehouse/models/mart/ridership/_mart_ridership.yml:1]: I think would be nice to show that are NTD data, using NTD on the group name like: `_mart_ntd_ridership`

File: warehouse/models/mart/annual_reporting/_mart_annual_reporting.yml
Comment (erikamov) [inline warehouse/models/mart/annual_reporting/_mart_annual_reporting.yml:1]: Same as the other groups, I think would be nice to show that are NTD data, using NTD on the group name like: `_mart_ntd_annual_reporting`

**Comment by vevetron** [comment]
I like the way these names are heading!

File: warehouse/models/mart/safety_and_security/_mart_safety_and_security.yml
Comment (charlie-costanzo) [reply warehouse/models/mart/safety_and_security/_mart_safety_and_security.yml:1]: pushed this change!

File: warehouse/models/mart/ridership/_mart_ridership.yml
Comment (charlie-costanzo) [reply warehouse/models/mart/ridership/_mart_ridership.yml:1]: pushed this change!

File: warehouse/models/mart/annual_reporting/_mart_annual_reporting.yml
Comment (charlie-costanzo) [reply warehouse/models/mart/annual_reporting/_mart_annual_reporting.yml:1]: pushed this change!

**Comment by vevetron** [comment]
If there are significant changes like this using code to rename etc, can you put a sample of that code into the PR description?

**Pull request merged**